lib/ansible/parsing/yaml/objects.py

```diff
@@ -19,9 +19,10 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import sys
 import yaml
 
-from ansible.module_utils.six import text_type
+from ansible.module_utils.six import text_type, PY2
```

Comment:
With the latest change this import is no longer needed:

```suggestion
from ansible.module_utils.six import text_type
```